mobilecoind/strategies/requirements.txt

```diff
@@ -1,22 +1,30 @@
 appnope==0.1.0
 backcall==0.1.0
-boto3==1.9
-botocore==1.4.93
+boto3==1.21.13
+botocore==1.24.13
+certifi==2021.10.8
+chardet==3.0.4
```

Comment:
Was the dependency update intentional?

Reply:
Short answer is yes. The main goal was getting grpcio up to date so I could use a pre-compiled pip package on Ubuntu 20.04. This of course ended up requiring a chain of updates and some new packages added.